packages/puppeteer-core/src/node/ProductLauncher.ts

```diff
@@ -203,12 +203,12 @@ export class ProductLauncher {
 
   executablePath(channel?: ChromeReleaseChannel): string;
   executablePath(): string {
-    throw new Error('Not implemented');
+    throw new UnsupportedOperation();
```

Comment:
I think this should be always implemented, abstract instead?

Reply:
Fair enough, we should switch the whole class to be `abstract`. Will do it in a separate PR.